distribution/lib/Standard/Base/0.0.0-dev/src/Network/Email/Email_Provider_SPI.enso

```diff
@@ -0,0 +1,38 @@
+import project.Data.Vector.Vector
+import project.Function.Function
+import project.Meta
+import project.Metadata.Display
+import project.Metadata.Widget
+from project.Metadata.Widget import Single_Choice
+from project.Nothing import all
+
+## ---
+   private: true
+   ---
+   Internal class for enabling SPI for Email_Provider types.
+type Email_Provider_SPI
+    private Value typ
+
+    ## ---
+       private: true
+       ---
+    new typ -> Email_Provider_SPI = Email_Provider_SPI.Value typ
```

Comment:
- this "factory method" is OK
- it should have two arguments `obj` and `dictionary` to match the changes in constructor